site/content/docs/monitoring/how-we-store-data.md

```diff
@@ -10,17 +10,21 @@ Every check we run generates a certain amount of data. We call these the "result
 
 - Metadata like timestamps, response times, fail/success flags, run location etc.
 - JSON or other response bodies, response headers, request timing data
+- Playwright Test traces and videos
 - Screenshot files
 - Run logs with debug or `console.log` statements.
 
 ## Data Retention
 
-We keep the results data according to the following retention policy:
+Depending on your plan we keep the results data according to the following retention policy:
 
-- Raw events for 30 days
-- 1 hour resolution forever
+|                               | Hobby            | Team             | Enterprise       |
+|-------------------------------|------------------|------------------|------------------|
+| **Raw data retention**        | 7                | 30               | 180              |
+| **Aggregated data retention** | 30               | 1 year           | 25 months        |
 
-Simply put, you can "scroll back" up to 30 days and inspect all the details of each result. After that, we store aggregates
+
+Simply put, you can "scroll back" within the defined period for raw data and inspect all the details of each result. After that, we store aggregates
 of the relevant statistics like average, p95 and p99 response time and success ratio.
 
 *We are continuously working on tweaking and improving our data retention policy. Expect updates.*
```

Comment:
@tnolet I updated the data retention section to reflect our current policy. Can this last sentence go?